data/ResourceUtil.cpp

```diff
@@ -1,104 +1,106 @@
-//this file was auto-generated by res2h
+// this file was auto-generated by res2h
 
 #include "Resources.h"
 
-const size_t res2hNrOfFiles = 46;
+const uint32_t res2hNrOfFiles = 47;
```

Comment:
NB: [I see](https://github.com/HorstBaerbel/res2h/commit/c7088db9b4d706a318e2aaa6a84f092caa43f1af) that rs2h was updated about 7 months ago, to enable 32bit and 64bit archives, which causes this type change.
I guess we need to make this transition at some point anyway, but it sure makes this review a bit unwieldy. My preference would be to do this in another, separate commit altogether, as it in itself has nothing to do with GridView.